pyquil/quil.py

```diff
@@ -768,7 +746,7 @@ def __add__(self, other: InstructionDesignator) -> "Program":
 
     def __iadd__(self, other) -> "Program":
         """
-        Concatenate two programs together by appending them to the left hand side.
+        Concatenate two programs together by appending them to the right-hand side to the left.
```

Comment:
```suggestion
        Concatenate two programs together by appending the right-hand side to the left.
```

I think this was my mistake